js/index.js

```diff
@@ -20,14 +20,68 @@ existingFooter.appendChild(copyright);
 const skills =["Javascript", "HTML", "CSS", "Adobe Photoshop", "gitHub"];
 
 const skillsSection = document.getElementById('skills');
-const skillsList = skillsSection.querySelector('u1');
+const skillsList = skillsSection.querySelector('ul');
 
 //For Loop
 for(let i =0; i<skills.length;i++){
     const skillItem  = document.createElement('li');
     skillItem.innerText = skills[i];
     skillsList.appendChild(skillItem);
 }
-//console.log('Skills Section:', skillsSection);
-//console.log('Skills List:', skillsList);
-// edit edit edit edit
+
+
+//message handler form submit
+const messageForm = document.forms["leave message"];
+
+messageForm.addEventListener("submit", function(event){
+    event.preventDefault();
+    
+//get values from the form fields
```

Comment:
Everything should be indented because it is inside the event callback function.

Reply:
I see that this is still not indented.  This makes the code difficult to follow.

Reply:
hopefully indented properly